### PR TITLE
Command improvements

### PR DESCRIPTION
**Pull request by sbcgua**
minor improvements on commands in package.json + commited package-lock

P.S. could you please publish the latest version on npm plz ?

P.P.S probably will try github actions on this repo if you don't mind :) (auto tag & npm publish)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 language: node_js
 node_js:
-  - "10"
+  - "12"
```
Comment (larshp) [inline .travis.yml]: 12 starts active LTS on 2019-10-22, https://github.com/nodejs/Release

there is nothing forcing us to update, would like to wait until at least vscode ships with node 12

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
 notifications:
   email: false
+script:
+  - npm run test
+  - npm run lint
+branches:
+  only: 
+    - master
```
Comment (larshp) [inline .travis.yml]: will this stop travis runing on pull requests?

File: package.json [hunk 3]
```diff
@@ -10,10 +10,10 @@
     "build": "tsc --pretty",
     "buildw": "tsc -w --pretty",
     "testw": "mocha --recursive --watch build/test",
-    "pretest": "tsc --pretty && mocha --recursive --reporter progress build/test",
-    "test": "tslint ./src/*.ts ./test/*.ts",
+    "pretest": "npm run build",
+    "test": "mocha --recursive --reporter progress build/test",
     "lint": "tslint ./src/*.ts ./test/*.ts",
-    "posttest": "browserify build/src/web.js -o web/bundle.js"
+    "browserify": "browserify build/src/web.js -o web/bundle.js"
```
Comment (larshp) [inline package.json]: browserify will not be run any longer?

File: .travis.yml (hunk 1 above)
Comment (sbcgua) [reply .travis.yml]: ok, will fix

File: .travis.yml (hunk 2 above)
Comment (sbcgua) [reply .travis.yml]: From what I remember no - there is a separete flag somewhere in travis which controls "pull requests to master", but let me double check, maybe I'm wrong

File: package.json (hunk 3 above)
Comment (sbcgua) [reply package.json]: well, now it's `npm run browserify` :) But this one is arguable, could you please tell what is the further flow with it ? What I meant is that browserify does not belong to test flow imho, so maybe it should be a part of another process.

File: .travis.yml (hunk 1 above)
Comment (sbcgua) [reply .travis.yml]: fixed

File: .travis.yml (hunk 2 above)
Comment (sbcgua) [reply .travis.yml]: re-checked - does not block PRs - controlled by this:
2019-09-17_00h29_42

File: package.json (hunk 3 above)
Comment (sbcgua) [reply package.json]: for this one waiting for your reply

**Comment by sbcgua** [comment]
let's merge maybe ? =)

**Pull request merged**